apps/linkedin-to-notion/package.json

```diff
@@ -10,13 +10,14 @@
     "package": "plasmo package"
   },
   "dependencies": {
+    "@notionhq/client": "^2.2.13",
     "@plasmohq/messaging": "0.5.0",
     "@plasmohq/storage": "1.7.2",
     "@supabase/supabase-js": "2.32.0",
-    "plasmo": "0.82.1",
+    "design-system": "workspace:*",
+    "plasmo": "0.83.0",
```

Comment:
You're not afraid 😅

Reply:
(fine with updating the version here but would be better to keep this for dedicated PR next time and make sure we know the diff between both versions)

Reply:
I now realize that I might have been slightly brazen ahaha